ops/ruby/devops-toolkit/devops.rb

```diff
@@ -26,6 +26,7 @@ def parse_options
     parser.on('-s', '--source-uri [SOURCE-URI]', String, 'URI or fully qualified name for source asset')
     parser.on('-u', '--source-env [SOURCE-ENV]', String, 'Short name for source Environment (lowercase), for example "staging".')
     parser.on('-d', '--dry-run', 'Execute a dry run of the task')
+    parser.on('-i', '--input-tools-file [INPUT-TOOLS-FILE]', String, 'Input YAML file for tools')
```

Comment:
so this is the option to be used for introducing a new yaml file to AoU ?

Reply:
it would be used for other projects. by default it uses the one I made for the aou project. The code doesn't care what project you're working on; it just takes them from the file.

Reply:
The AoU input file (currently the only one) was introduced in #3243 .